plugins/catalog/src/alpha/pages.tsx

```diff
@@ -62,21 +65,57 @@ export const catalogEntityPage = PageBlueprint.makeWithOverrides({
       EntityContentBlueprint.dataRefs.title,
       EntityContentBlueprint.dataRefs.filterFunction.optional(),
       EntityContentBlueprint.dataRefs.filterExpression.optional(),
+      EntityContentBlueprint.dataRefs.group.optional(),
     ]),
   },
-  factory(originalFactory, { inputs }) {
+  config: {
+    schema: {
+      groups: z =>
+        z.array(z.object({ id: z.string(), title: z.string() })).optional(),
+    },
+  },
+  factory(originalFactory, { config, inputs }) {
     return originalFactory({
       defaultPath: '/catalog/:namespace/:kind/:name',
       routeRef: convertLegacyRouteRef(entityRouteRef),
       loader: async () => {
-        const { EntityLayout } = await import('../components/EntityLayout');
+        const { EntityLayout } = await import('./components/EntityLayout');
+
+        // Config groups can override default group titles
+        const groups: Record<string, string> = {
+          ...defaultEntityContentGroups,
+          ...config.groups?.reduce(
+            (rest, group) => ({ ...rest, [group.id]: group.title }),
+            {},
+          ),
+        };
+
+        const tabs = inputs.contents.reduce<
```

Comment:
Does it make sense to show groups in the order their entity contents appear?

Reply:
Probably not tbh, but leaving this as a thing to potentially tweak